scripts/nayduck.py

```diff
@@ -188,10 +193,11 @@ def run_locally(tests):
 
         ignored = fields[1:index]
         del fields[1:index]
-        message = f'Running ‘{"".join(fields)}’'
-        if ignored:
-            message = f'{message} (ignoring flags ‘{"".join(ignored)}`)'
-        print(message)
+        if not args.dry_run:
+            message = f'Running ‘{"".join(fields)}’'
+            if ignored:
+                message = f'{message} (ignoring flags ‘{"".join(ignored)}`)'
```

Comment:
Since you’re touching those, perhaps also fix `" ".join(…)` in both places.

Reply:
Ok, fixed.